_posts/2022-09-29-r-text/r-text.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+title: "The text-package"
+description: |
+  The [text](http://r-text.org/)-package attempts to provide user-friendly access and pipelines to HuggingFace's transformer language models in R.
+author:
+  - name: Oscar Kjell 
+    url: https://oscarkjell.se/
+    affiliation: Lund University
+    affiliation_url: https://portal.research.lu.se/en/persons/oscar-kjell
+  - name: Salvatore Giorgi 
+    url: http://www.salvatoregiorgi.com/
+    affiliation: University of Pennsylvania
+    affiliation_url: http://www.salvatoregiorgi.com/
+  - name: H Andrew Schwartz  
+    url: https://www3.cs.stonybrook.edu/~has/
+    affiliation: Stony Brook University
+    affiliation_url: https://www3.cs.stonybrook.edu/~has/
+categories:
```

Comment:
Please only make use of existing categories :-)
From these, only 

Natural Language Processing 

exists. Plus, we have 

R

as a category always (if R-related), for a technical reason.

Thanks!

Reply:
removed, and included R